src/stores/business.ts

```diff
@@ -15,7 +15,7 @@ export const useBcrosBusiness = defineStore('bcros/business', () => {
   const currentBusinessIdentifier = computed((): string => currentBusiness.value.identifier)
   const currentBusinessName = computed((): string => {
     if (currentBusiness.value.alternateNames && currentBusiness.value.legalType === CorpTypeCd.SOLE_PROP) {
-      return currentBusiness.value.alternateNames[0].operatingName
+      return currentBusiness.value.alternateNames[0].name
```

Comment:
This actually needs to be updated to be: `currentBusiness.value.alternateNames.find(name => name.identifier === currentBusinessIdentifier.value) || currentBusiness.value.legalName` because there are cases of multiple 'alternate names' and the correct one for 'operating name' has the matching identifier

Also it needs to do this for GPs as well

Reply:
Updated, whats the reason why would other alternate names have different identifiers ?

Reply:
I don't really know all the rules surrounding it, but there are many cases (went through this with Argus a couple weeks ago before they released it into prod and he said I could always find the right one by the identifier match)

Reply:
Just need to add this for GPs as well

Reply:
oh nvm you added it!